legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/analyticsHelper.pure

```diff
@@ -43,7 +51,14 @@ function meta::analytics::mapping::modelCoverage::getRealProperty(property:Abstr
 
 function <<access.private>> meta::analytics::mapping::modelCoverage::buildPropertyInfo(p:AbstractProperty<Any>[1]):PropertyInfo[1]
 {
-   ^PropertyInfo(name = $p->originalPropertyNameForEdgePointProperty()->toOne(), multiplicity = $p.multiplicity);
+   let name = $p->originalPropertyNameForEdgePointProperty()->toOne();
+   ^PropertyInfo(
+      taggedValues = $p.taggedValues->map(val | ^TaggedValueInfo(tagOwner = $val.tag.profile->elementToPath(), tagValue = $val.tag.value, value = $val.value)),
```

Comment:
would be nice to make this a function that takes in taggedValues so can be reused below. same for stereotypes